pkg/utils/utils.go

```diff
@@ -99,3 +99,7 @@ func GuessMimeType(key string) string {
 	}
 	return mimeType
 }
+
+func EncodeUserInput(input string) string {
```

Comment:
RemoveNewLine().

It seems that this is only used by pkg/fs, we should put it there.